app/models/etmp/EtmpSchemeDetails.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.etmp
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.format.DateTimeFormatter
+
+case class EtmpSchemeDetails(commencementDate: String,
+                             firstSaleDate: Option[String],
+                             euRegistrationDetails: Option[Seq[EtmpEuRegistrationDetails]],
+                             previousEURegistrationDetails: Option[Seq[EtmpPreviousEURegistrationDetails]],
```

Comment:
For both the sequences, we shouldn't make them optional, I know they're optional in the schema, but the model should allow empty sequence

Reply:
Changed both from option[Seq] to Seq